about.html

```diff
@@ -0,0 +1,172 @@
+            </ul>
+          </div>
+        </nav>
+      </header>
+    </main>
+    <!-- About section -->
+    <section class="about-page fonts">
+      <p class="a-hello">"Impossible is nothing!"</p>
+      <h1 class="about-title fonts-coco">
+        RMsoft.ltd - Reliable Made Software
+      </h1>
+      <div class="a-content">
+        <p>
+          RMsoft.ltd company (Reliable Made Software) founded in 2012 by a team
+          of professionals who are passionate about delivering the best web
+          based software, android applications, Cabling solutions, CCTV and
+          Access control solutions. Since 2012, we have developed systems like
+          MEDSTORE, SUPERSTORE, Rwanda Neuro-Surgical Center content management
+          system, TOEFL correction software and many more that deliver ultimate
+          digital experiences for financial services and insurance companies,
+          NGOs and government agencies.
+        </p>
+      </div>
+      <p class="a-copyright">Get in touch with us!</p>
+      <a class="email" href="mailto: rushemaa@gmail.com">rushemaa@gmail.com</a>
+    </section>
+    <!-- still about  -->
+    <section class="a-section-1 fonts">
+      <div class="a-section-1-title">
+        <h2>The RMsoft company Logo</h2>
+        <hr class="a-line" />
+      </div>
+      <p class="logo-desc">
+        The logo of was designed by one of our best UI designer in 2015
+      </p>
+
+      <div class="a-section-1-logo">
+        <img class="a-logo" src="images/rm.jpg" />
+      </div>
+    </section>
+    <!-- still about  -->
+    <section class="a-section-1 sec-2 fonts">
+      <div class="a-section-1-title">
+        <h2>Best selling Projects</h2>
+        <hr class="a-line" />
+      </div>
+      <p class="logo-desc">
+        Below are our best selling projects in the last 5 years
+      </p>
+      <div class="past-events">
+        <div class="a-section-1-logo events">
+          <div class="event-rapper">
+            <h2 class="h-events">2018</h2>
+            <p>MEDSTORE</p>
+          </div>
+        </div>
+        <div class="a-section-1-logo events-1">
+          <div class="event-rapper">
+            <h2 class="h-events">2020</h2>
+            <p>KIGALISM.COM</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- still about  -->
+    <section class="partners-wrapper fonts">
+      <div class="partner-title">
+        <h2>Parter</h2>
+        <hr class="partner-line" />
+      </div>
+
+      <div class="partners">
+        <img class="partner" src="images/rdb.jpeg" />
+        <img class="partner" src="images/oda.png" />
+        <img class="partner" src="images/rwa.png" />
+      </div>
+    </section>
+    <!-- About last section -->
+    <section class="footer fonts">
+      <h3>&#169; RMsoft</h3>
+      <p class="footer-c">
+        2023 Rushema Anicet Kigali<br />
+        Some Rights Reserved
+      </p>
+    </section>
```

Comment:
- [ ] Kindly place this container to the center of the page. so as to have a very pixel perfect design 😉.